src/script/components/MessagesList/Message/ContentMessage/Warnings/PartialFailureToSend/PartialFailureToSend.tsx

```diff
@@ -129,7 +129,7 @@ export const PartialFailureToSendWarning = ({failedToSend, knownUsers}: Props) =
 
               {/* maps through the unreachable users that will never receive the message:
               "3 participants from alpha.domain, 1 participant from beta.domain won't get your message" */}
-              {failed && (
+              {unreachableUsers.length !== 0 && (
```

Comment:
How about adding a test case for this one ;) (check that this message doesn't display if there aren't any unreachable)?